### PR TITLE
Create User Input class to refactor user_place_ship method

### PR DESCRIPTION
**Pull request by mkrumholz**
## What does this change?

Creates UserInput class

## What does this fix?

Allows us to refactor methods with user input and enable testing

## Is this a bug or a feature? 

feature

## How was this tested?

Tested using runner file

### CODE REVIEW

File: lib/setup.rb [hunk 1]
```diff
@@ -76,15 +73,20 @@ def list_ships(ships)
       ships.each do |ship|
       puts @messages.ship_details(ship)
       end
-    end 
+    end
 
-    def get_valid_placement(ship, coordinates) 
+    def get_valid_placement(ship)
+      coordinates = get_multiple_coordinates
       while board.valid_placement?(ship, coordinates) == false
```
Comment (suzkiee) [inline lib/setup.rb:80]: Should this be a helper method?

File: lib/setup.rb [hunk 2]
```diff
@@ -76,15 +73,20 @@ def list_ships(ships)
...
-        print "> "
-        user_input = gets.chomp.upcase
-        puts @messages.new_line
-        coordinates = user_input.split(' ')
+        coordinates = get_multiple_coordinates
       end
+      coordinates
+    end
+
+    def get_multiple_coordinates
```
Comment (suzkiee) [inline lib/setup.rb:87]: GREAT refactor on this, Molly!

**Comment by mkrumholz** [comment]
## What does this change?

Creates UserInput class

## What does this fix?

Allows us to refactor methods with user input and enable testing

## Is this a bug or a feature? 

feature

## How was this tested?

Tested using runner file

File: lib/user_input.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class UserInput
```
Comment (suzkiee) [inline lib/user_input.rb:1]: This is beautiful

File: lib/setup.rb (hunk 1 above)
Comment (mkrumholz) [reply lib/setup.rb:80]: Yes, let's do this later!

**Pull request merged**